contrib/digitalocean/README.md

```diff
@@ -1,3 +1,23 @@
 # Provision a Deis Cluster on DigitalOcean
 
-Please refer to the instructions at http://docs.deis.io/en/latest/installing_deis/digitalocean/.
+Using Rigger:
+
+    ./rigger configure
+      # answer "Digital Ocean when asked what provider you want to use"
+      # you'll also need to provide your Digital Ocean token and SSH fingerprint
+
+    ./rigger provision
+      # use your new Deis cluster when this command completes!
+
+    ...
+
+    ./rigger destroy
+      # when done with the cluster
+
+Using Terraform:
+
+    terraform apply
+      # will ask you some questions to fill in variables
+      # use your new Deis cluster when this command completes!
+
+Then follow the instructions on [provisioning Deis on Digital Ocean manually](http://docs.deis.io/en/latest/installing_deis/digitalocean/)
```

Comment:
Why wouldn't we include the `rigger` commands in the DigitalOcean instructions at docs.deis.io? 

This README.md is intentionally just a pointer to that, so that there is one source of truth, and also so content isn't split between markdown and restructuredtext.